scala-package/mxnet-demo/java-demo/README.md

```diff
@@ -1,13 +1,15 @@
 # MXNet Java Sample Project
 This is an project created to use Maven-published Scala/Java package with two Java examples.
 ## Setup
-You can use the `Makefile` to make the Java package. Simply do the following:
-```Bash
-make javademo
+User are required to use maven to build the package, by simply running
+```
+mvn package
 ```
-This will load the default parameter for all the environment variable.
-If you want to run with GPU on Linux, just simply add `USE_CUDA=1` when you run the make file
- 
+This command will pick the default values specified in the pom file.
+
+Note: If you are planning to use GPU, please add `-Dmxnet.profile=linux-x86_64-gpu`
+
+### Use customized version set
 You can use the following instruction as an alternative to achieve the same result:
 User are required to use `mvn package` to build the package,
```

Comment:
Grammar on this line... also is this true? Should it be you "may" not "must" or "required"?
```suggestion
You are required to use `mvn package` to build the package,
```